client/state/geolocation/actions.js

```diff
@@ -0,0 +1,32 @@
+import superagent from 'superagent';
```

Comment:
Conventionally we add `External dependencies` and `Internal dependencies` docblocks.